go/executor/executor.go

```diff
@@ -42,7 +42,7 @@ import (
 	pb "sqlflow.org/sqlflow/go/proto"
 )
 
-var rePyDiagnosis = regexp.MustCompile("runtime.tensorflow.diag.SQLFlowDiagnostic: (.*)")
+var rePyDiagnosis = regexp.MustCompile("runtime.diag.SQLFlowDiagnostic: (.*)")
```

Comment:
As the comment https://github.com/sql-machine-learning/sqlflow/pull/2682/files#r454953210 , rename `diag` to a full name `diagnostic` can be more clear.

Reply:
Done.